### DISCUSSION TITLE
Websocket response interceptor?

### DISCUSSION THREAD

**Discussion by sbreiler-work** [chimurai/http-proxy-middleware#635] [Q&A]
Hi! Is it possible to change the repsonse of an websocket reply? _onProxyRes_ seems to be only available for http streams.

**Comment by chimurai** [comment]
Not sure if it's possible.

Don't see an option to listen to WS responses in https://www.npmjs.com/package/http-proxy
Unless I saw it incorrectly.

**Reply by sbreiler-work** [reply]
Thanks for the quick reply.

I'm pretty shure, http-proxy doesn't natively support modifying WS response - that's why i hoped you found a way :-)
